src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -240,6 +257,7 @@ public Result<Void> isValidStatus() {
 
   public Result<Void> openLoanHasUserId() {
     if (isOpen() && getUserId() == null) {
+      log.warn("openLoanHasUserId:: Open loan must have a user ID");
```

Comment:
Can we extract message into a variable and reuse it?

Reply:
done